dev/com.ibm.ws.jca.cm/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -28,6 +28,9 @@ connectionManager.desc=Connection Manager configuration
 agedTimeout=Aged timeout
 agedTimeout.desc=Amount of time before a connection can be discarded by pool maintenance. A value of -1 disables this timeout. A value of 0 discards every connection, which disables connection pooling.
 
+abortLongRunningInuseConnections=Abort long running in use connections
+abortLongRunningInuseConnections.desc=The amount of time before an in-use connection can be aborted by pool maintenance.  A value of -1 or 0 disables this timeout. Aborting connections that are in use can result in unexpected behaviors,  The recommended value is 20 minutes or more depending on the applications normal transaction times.  If transaction times are high, 10 minutes or more, then double your longest running transaction time for this value.
```

Comment:
Suggested change to remove "abort," fix spacing, etc.:

The amount of time before pool maintenance can stop an in-use connection. A value of -1 or 0 disables this timeout. Stopping connections that are in use can result in unexpected behaviors. The recommended value is 20 minutes or more depending on the normal application transaction times. If transaction times are high, 10 minutes or more, then double your longest running transaction time for this value.